rules/config.smk

```diff
@@ -148,41 +106,41 @@ def resolve_config_value(*rule_parts, sep="/"):
 
     return resolve
 
-def resolve_config_path(*fields):
+def resolve_config_fields_path(*fields):
     """
     A helper function intended to be used as directly as a Snakemake Input
     function to resolve the appropriate config path.
 
     Given an array of config *fields* (keys), we return a function with a single
     argument *wildcards* which resolves to a appropriate local file path
     (string). These are accomplished via calls to helper functions
-    `resolve_config_value` and `resolve_path`, respectively; see the docstrings
+    `resolve_config_value` and `resolve_config_path`, respectively; see the docstrings
```

Comment:
Regardless of the ongoing naming discussions, we should add that `resolve_config_path` comes from the shared ruleset.

Reply:
Updated in https://github.com/nextstrain/avian-flu/pull/150/commits/35d3b9ea29a90315af5ced81d2503b9bab24ab72